meaningfulconsent/templates/main/participant_sessions.html

```diff
@@ -49,6 +49,7 @@
                     <td>
                     <div class="pull-right">
                         <form action="/participant/login/" method="post" style="display: inline-block">
+                            <input type="hidden" name="csrfmiddlewaretoken" value="<%=csrfmiddlewaretoken%>">
```

Comment:
Ohh I see -- you're getting this token from the client-side code.

From what I can tell, you're just getting the cookie, though?